examples/hubert/dataset/hubert_dataset.py

```diff
@@ -250,7 +250,7 @@ def __init__(
         self.pad = pad
         self.rand_crop = rand_crop
 
-    def __call__(self, batch: Tuple[Tensor, Tensor, int]) -> Tuple[Tensor, Tensor, Tensor]:
+    def __call__(self, batch: List[Tuple[Tensor, Tensor, int]]) -> Tuple[Tensor, Tensor, Tensor]:
```

Comment:
Is this bugfix or spec change?

Reply:
It's a bug fix. The batch should be a list of tuples instead of one tuple.